.travis.yml

```diff
@@ -16,10 +16,8 @@ install:
   - composer install
 
 before_script:
-  - export WEB_FIXTURES_HOST=http://localhost:8000
-
   # Start a webserver for web fixtures. Force using PHP 5.6 to be able to run it on PHP 5.3 and HHVM jobs too
-  - ~/.phpenv/versions/5.6/bin/php -S localhost:8000 -t vendor/behat/mink/driver-testsuite/web-fixtures > /dev/null 2>&1 &
+  - MINK_PHP_BIN=~/.phpenv/versions/5.6/bin/php vendor/bin/mink-test-server > /dev/null 2>&1 &
```

Comment:
Then splitting variable definition into separate line would make this fact more obvious. If you do so, then please update other PRs with similar construct in `.travis.yml`.